styleguide/components/21-atoms/inputs/input-multiselect-tagging/input-multiselect-tagging.bindings.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * Javascript binding of baguettebox.min.js.
+ */
+'use strict';
+
+(function () {
+
+  var elements = document.getElementsByClassName('multiselect-tagging'); // eslint-disable-line no-undef
```

Comment:
We have babel now (well... I think it's still in my branch though...)

Reply:
Ok, so good for now.